blocks/byline-block/intl.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
Removing this file will require updates to the code in this block to ensure it's pull the translation by the new key names now

Reply:
updated

Reply:
I pushed an updated - as the byline block intl.json file is manually curated and the values are not part of any locale main file

Reply:
nice